### PR TITLE
Incorporate new DB structure for persons and affiliations

### PR DESCRIPTION
**Pull request by wow-such-code**
- remove phone number
- organization name from affiliation table is displayed for project manager, principal investigator and other persons on the project info page instead of some form of information from organizations table

### CODE REVIEW

**Comment by wow-such-code** [comment]
[Enable ProjectBrowser to use new person table(s)](https://trello.com/c/RMPGAhIu/711-enable-projectbrowser-to-use-new-person-tables)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,13 @@
 ==========
 Changelog
 ==========
+1.11.4 (2021-03-22)
```
Comment (sven1103) [inline CHANGELOG.rst:4]: I think it is `1.11.3` isn't it?

File: src/main/java/life/qbic/projectbrowser/components/LevelComponent.java [hunk 2]
```diff
@@ -992,6 +985,34 @@ public void registerDatasetInTable(DatasetBean d, HierarchicalContainer dataset_
...
+        blackList.add(childId);
+      }
+    }
+    for (Object rowId : datasetTable.getItemIds()) {
+      if (!blackList.contains(rowId) && !isParentOf(rowId, itemId)) {
+        CheckBox itemCheckBox =
+            (CheckBox) datasetTable.getItem(rowId).getItemProperty("Select").getValue();
+        itemCheckBox.setValue(false);
+      }
+    }
```
Comment (sven1103) [inline src/main/java/life/qbic/projectbrowser/components/LevelComponent.java:1004]: These changes are unrelated to the PR. Is the development branch out of sync?

**Review by sven1103 — CHANGES_REQUESTED** [review]
Thank you @wow-such-code for this contribution. It looks like the feature branch is ahead of the development branch and introduces changes unrelated to the PR.
Can you please update development based on any hotfixes not available yet?

File: CHANGELOG.rst (hunk 1 above)
Comment (wow-such-code) [reply CHANGELOG.rst:4]: 1.11.4 is on production, I think either 1.11.3 was broken or we forgot to document the changes

**Comment by wow-such-code** [comment]
should be fixed now

**Comment by wow-such-code** [comment]
[Update Project browser to deal with the new user db](https://trello.com/c/HzvA2dK9/756-update-project-browser-to-deal-with-the-new-user-db)
